scm/etc/scripts/plot_configs/twpice_all_suites.ini

```diff
@@ -15,7 +15,7 @@ time_series_resample = True
 
 [plots]
   [[profiles_mean]]
-    vars = qc, qv, T, dT_dt_PBL, dT_dt_conv, dT_dt_micro, dT_dt_lwrad, dT_dt_swrad
```

Comment:
The plotting script fails with these convective tendencies when SAS is not included. These are added back into the plot configuration in the online tutorial when SAS is added.